client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-add-license-notification/wpcom-atomic-hosting-notification.tsx

```diff
@@ -0,0 +1,88 @@
+import { Button, Gridicon } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import checkIcon from 'calypso/assets/images/jetpack/jetpack-green-checkmark.svg';
+import Banner from 'calypso/components/banner';
+import { useDispatch } from 'calypso/state';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import { setPurchasedLicense } from 'calypso/state/jetpack-agency-dashboard/actions';
+import type { PurchasedProductsInfo } from '../types';
+
+export default function WPCOMAtomicHostingNotification( {
+	licensesAdded,
+}: {
+	licensesAdded: PurchasedProductsInfo;
+} ) {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const { selectedSite, selectedProducts } = licensesAdded;
+
+	const licenseItem = selectedProducts[ 0 ].name;
+
+	const handleOnClick = ( eventName: string ) => {
+		dispatch(
+			recordTracksEvent( eventName, {
+				site: selectedSite,
+				licenseItem,
+			} )
+		);
+		dispatch( setPurchasedLicense() );
+	};
+
+	const extraContent = (
+		<div className="site-add-license-notification__license-banner-wp-buttons">
+			<Button
+				href={ `https://wordpress.com/home/${ selectedSite }` }
+				primary
+				target="_blank"
+				onClick={ () =>
+					handleOnClick( 'calypso_jetpack_agency_dashboard_setup_site_in_wp_admin_click' )
+				}
+			>
+				{ translate( 'Setup site in wp-admin' ) }
+				<span>
+					<Gridicon icon="external" size={ 18 } />
+				</span>
+			</Button>
+
+			<Button
+				borderless
+				target="_blank"
+				href={ `https://wordpress.com/domains/manage/${ selectedSite }` }
```

Comment:
Just checking here, since it's a bug that came up recently: is `selectedSite` is a slug here, rather a URL?

Reply:
Oh, yes. Good catch. I have made the changes: https://github.com/Automattic/wp-calypso/pull/81173/commits/a14a2af29983355f19a3d8da28314a35a46fde11